src/index.ts

```diff
@@ -3,3 +3,9 @@ import './configure';
 
 export * from './types';
 export * from './components';
+
+export {createThemeContext} from './utilities/theme-provider';
```

Comment:
Exports to keep these in our bundle

Reply:
I mostly added this because we already exported it. I'm a little iffy on removing our exports from `src/components`. We export everything from `src/components` which I think is coming back to bite us. I'm the future we should definitely not export every single component we have, and selectively export our public API. Technically IMO this is undocumented, and not part of our public API so we can safely remove it, but we do export it. So it could also be seen as a breaking change. What are you thoughts?

To answer your question theme context can be used "independently" from polaris-context but ultimately this is nothing `createPolarisContext` won't be able to handle and could be 🔥 from our exports

Reply:
This is v4, we're free to make breaking changes :)

Looking at https://github.com/search?p=3&q=org%3AShopify+createThemeContext&type=Code It seems like there's a few usages on createThemeContext() by consuming projects in testing setups (next to AppProviderContext) but they can all be replaced with `createPolarisContext()`. so that feels like the api we should be promoted going forward.

I think remove this and tell people using this that they should use `createPolarisContext()` instead.

Reply:
> This is v4, we're free to make breaking changes :)

Anything we can do to reduce the surface area of our API makes sense to do here. If consumers don't _really_ need it, don't export it.